recycler.sh

```diff
@@ -254,6 +254,26 @@ recreate_volume() {
   return 0
 }
 
+# Verifies that no fuse clients are connected to the volume given
+#
+# $1 - list of gluster endpoints
+# $2 - name of the volume
+volume_is_used() {
+  local rhost="${1%%,*}"  # extract the first entry
+  local vol_name="$2"
+  nfuse="$(
+    gluster --remote-host="$rhost" --mode=script \
+      volume status "$vol_name" client-list | \
+    awk '/^fuse\s+[[:digit:]]+$/{print $2}')"
```

Comment:
This will ignore any non-FUSE clients. What's the rationale for not counting all clients except `glustershd`?

Reply:
TODO: Check what the docs have to say about possible outputs; change code to count non-glustershd-clients

Reply:
Reimplemented `volume_is_used` to handle all types of clients (as long as they match `\w+`)